core/src/main/java/org/apache/carbondata/core/metadata/datatype/ArrayType.java

```diff
@@ -65,4 +73,11 @@ public DataType getElementType() {
     return elementType;
   }
 
+  public String getElementName() {
+    return elementName;
+  }
+
+  public void setElementName(String elementName) {
```

Comment:
Please remove if not used

Reply:
should keep base method

Reply:
even base class dont have `setmethod` for `name`. can remove it

Reply:
removed